easybuild/easyconfigs/o/OpenCV/OpenCV-4.5.1-foss-2020b-contrib.eb

```diff
@@ -37,6 +37,8 @@ dependencies = [
     ('SciPy-bundle', '2020.11'),  # for numpy
     ('zlib', '1.2.11'),
     ('FFmpeg', '4.3.1'),
+    ('freetype', '2.10.4'),
```

Comment:
@guimou We have an easyconfig file for freetype 2.10.3 for this generation of easyconfigs (2020b, so `freetype-2.10.3-GCCcore-10.2.0.eb` will be picked up), so please change this to `'2.10.3'`, which should fix the failing tests

Reply:
Oh, yes, my bad! It was a remnant from a test when I was trying to figure it our, I created a package with the latest version in my dev environment. Corrected.